### PR TITLE
Add documentation for new environment variables

### PR DESCRIPTION
**Pull request by ccremer**
In PR #15 a bunch of variables are now configurable with ENV vars, but not further documented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,21 @@ Defines how the tunnel will be set up:
...
+#### SERVER_ALIVE_INTERVAL, SERVER_ALIVE_COUNT_MAX
+
+From `man ssh_config` (https://linux.die.net/man/5/ssh_config) (2020-02-21):
+
+> ServerAliveInterval
+  Sets a timeout interval in seconds after which if no data has been received from the server, ssh(1) will send a message through the encrypted channel to request a response from the server. The default is 0, indicating that these messages will not be sent to the server [...]
+
+> ServerAliveCountMax
+  Sets the number of server alive messages (see below) which may be sent without ssh(1) receiving any messages back from the server. If this threshold is reached while server alive messages are being sent, ssh will disconnect from the server, terminating the session. [...] The server alive mechanism is valuable when the client or server depend on knowing when a connection has become inactive.
+  The default value is 3. If, for example, ServerAliveInterval (see below) is set to 15 and ServerAliveCountMax is left at the default, if the server becomes unresponsive, ssh will disconnect after approximately 45 seconds.
```
Comment (ccremer) [inline README.md]: Not sure if we want to quote the parameters or just link to the docs. WDYT?

**Comment by ccremer** [comment]
Updated.

Sadly I do not have any more examples. Our customer is still somewhat trying out the best configuration options.

File: README.md [hunk 2]
```diff
@@ -153,6 +153,25 @@ Defines how the tunnel will be set up:
...
+
+`SERVER_ALIVE_COUNT_MAX` sets the threshold of alive messages after which the connection is
+terminated and reestablished. `3` is the default
+
+Additional details are available from [`ssh_config(5)`](https://linux.die.net/man/5/ssh_config)
+
+#### Additional Environment variables
+
+* From [`autossh(1)`](https://linux.die.net/man/1/autossh)
+* From [`ssh_config(5)`](https://linux.die.net/man/5/ssh_config)
```
Comment (jnovack) [inline README.md]: Don't need "From", the list item is not meant to be a sentence.

File: README.md [hunk 3]
```diff
@@ -170,44 +189,45 @@ on the private LAN of the docker host.  `ssh`ing to fake internet address
...
-          - SSH_TUNNEL_HOST=198.168.123.45
-          - SSH_TUNNEL_LOCAL=22
-        restart: always
-        volumes:
-          - /etc/autossh/id_rsa:/id_rsa
-        dns:
-          - 8.8.8.8
-          - 4.2.2.4
-
+```yaml
```
Comment (jnovack) [inline README.md]: Heh, thanks.

**Comment by ccremer** [comment]
Another round?

**Pull request merged**